test/cli/test_delete.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from unittest.mock import patch, MagicMock
+from click.testing import CliRunner
+from ersilia.cli.commands.delete import delete_cmd
+from ersilia.utils.logging import logger
+
+@pytest.fixture
+def runner():
+    return CliRunner()
+
+MODEL_ID = "eos3b5e"
+@patch("ersilia.core.modelbase.ModelBase")
+@patch("ersilia.hub.delete.delete.ModelFullDeleter.needs_delete", return_value=None)
```

Comment:
We should divide this into two functions: when the model is available, where the return value from this mock would be 'True', and then the other case where model is not available, making the return value from the mock 'False', which would change the output message from the mock for the assertion.

Reply:
Resolved, but found some bug @DhanshreeA , that even the model is not there it consider it as it existed and requires deletion (this function `needs_delete`). So for now could not create the second function. But this current change the general health of this `cmd`.